tests/apollo/test_skvbc_persistence.py

```diff
@@ -345,7 +345,7 @@ async def test_st_when_fetcher_and_sender_crash(self, bft_network, tracker):
 
         client, known_key, known_val = \
             await skvbc.prime_for_state_transfer(stale_nodes={stale_node},
-                                                           checkpoints_num=random.randint(10, 13))
+                                                 checkpoints_num=10)
```

Comment:
can you explain this change?

Reply:
wanted to be able to replicate the test's behavior and compare logs